lib/state/state_container.dart

```diff
@@ -43,6 +44,12 @@ class StateContainerState extends State<StateContainer> {
         ..isLoading = state.isLoading);
     });
   }
+
+  void setGameStatuses(BuiltMap<String, GameStatus> gameStatuses) {
```

Comment:
we dont need to set a bunch of game statuses. Once a game is played, we might have to set the status for an individual game